### PR TITLE
fix: update combobox design to latest spec

### PR DESCRIPTION
**Pull request by Jinnie**
Also fixes a regression on the invalid state notification

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [x] Other... Please describe: Update to latest design spec

## What is the current behavior?

Currently the pills are designed to design spec minus one - based on the labels styles
Issue Number: N/A

## What is the new behavior?

New style adheres to latest specs from design team, with smaller rounded borders, overall component border, different color scheme.

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by bdryanovski — APPROVED** [review]
LGTM

File: packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss [hunk 1]
```diff
@@ -12,9 +12,12 @@ $clr-combobox-trigger-left-padding: $clr_baselineRem_0_25 !default;
...
 
 $clr-combobox-font-size: $clr-p2-font-size !default;
+$clr-combobox-border-color: hsl(198, 10%, 90%) !default;
+$clr-combobox-border-radius: $clr_baselineRem_3px !default;
 
-$clr-combobox-input-background-color: hsl(201, 10%, 96%) !default;
-$clr-combobox-pill-background-color: hsl(201, 80%, 95%) !default;
-$clr-combobox-pill-border-color: hsl(201, 55%, 81%) !default;
+$clr-combobox-input-background-color: hsl(198, 12%, 97%) !default;
+$clr-combobox-pill-background-color: hsl(198, 0%, 100%) !default;
```
Comment (mathisscott) [inline packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss:19]: This is `$clr-color-neutral-0`

File: packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss [hunk 2]
```diff
@@ -12,9 +12,12 @@ $clr-combobox-trigger-left-padding: $clr_baselineRem_0_25 !default;
...
+$clr-combobox-pill-border-color: hsl(198, 10%, 90%) !default;
```
Comment (mathisscott) [inline packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss:20]: Why is design giving us so many off-palette colors?

**Comment by mathisscott** [comment]
@Jinnie 
I'm assuming this won't be going into tomorrow's v4 release. That's fine, if so. Just verifying.

**Comment by Jinnie** [comment]
> @Jinnie
> I'm assuming this won't be going into tomorrow's v4 release. That's fine, if so. Just verifying.

No, I'll get the feedback from design team tomorrow, then I will need to update the site with the new screenshots and port it to v4. I don't think we can do it for tomorrow/today.

File: packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss (hunk 1 above)
Comment (Jinnie) [reply packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss]: done

File: packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss (hunk 2 above)
Comment (Jinnie) [reply packages/angular/projects/clr-angular/src/forms/combobox/_variables.combobox.scss]: After pairing with design on the problem, we changed all these to Clarity palette colors ($clr-color-..............).
Colors in the dark theme were updated to match those from the Accordion component, which features similar levels of shading as the Combobox. We don't have pre-defined dark theme palette, but this should be addressed separately from this issue.
Caret color was updated to text color, for better color contrast.

**Pull request merged**